streaming/streaming.go

```diff
@@ -459,9 +459,52 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 			dataflowSubnetwork = fmt.Sprintf("https://www.googleapis.com/compute/v1/projects/%s/regions/%s/subnetworks/%s", dataflowHostProjectId, dataflowCfg.Location, dataflowCfg.Subnetwork)
 		}
 	}
-
-	launchParameters := createLaunchParameters(dataflowCfg, inputFilePattern, project, datastreamCfg, instance, dbName, streamingCfg, dataflowSubnetwork, workerIpAddressConfig)
-
+	fmt.Printf("Working with dfCfg: %v\n", dataflowCfg)
+	maxWorkers := int32(50)
+	numWorkers := int32(1)
+	if dataflowCfg.MaxWorkers != "" {
+		intVal, err := strconv.ParseInt(dataflowCfg.MaxWorkers, 10, 64)
+		if err != nil {
+			return fmt.Errorf("could not parse MaxWorkers parameter %s, please provide a positive integer as input", dataflowCfg.MaxWorkers)
+		}
+		maxWorkers = int32(intVal)
+		if maxWorkers < 1 || maxWorkers > 1000 {
+			return fmt.Errorf("maxWorkers should be a positive integer (> 0) that does not exceed 1000")
+		}
```

Comment:
`1` and `1000` should be declared as global constants so that its easy to understand what does the check mean